srcStatic/OP2Memory.cpp

```diff
@@ -110,7 +110,7 @@ bool Op2RelinkCall(std::size_t callOffset, void* newFunctionAddress)
 
 	// Verify this is being run on a CALL instruction
 	if (*reinterpret_cast<unsigned char*>(callOffset) != 0xE8) {
-		PostErrorMessage("OP2Memory", __LINE__, "Op2RelinkCall error: No CALL instruction found at given address: " + AddrToHexString(callOffset));
+		PostErrorMessage(__FILE__, __LINE__, "Op2RelinkCall error: No CALL instruction found at given address: " + AddrToHexString(callOffset));
```

Comment:
Thanks for fixing my mistake here using the hard-coded value instead of the __FILE__ macro. I had forgotten about __FILE__.